### PR TITLE
Add Trino Fest recap blog post

### PR DESCRIPTION
**Pull request by mosabua**
Numbers are confirmed by Starburst event team. Video and other links are all tested.

### CODE REVIEW

**Comment by mosabua** [comment]
Please also read and review @ryandoowan

File: _posts/2023-06-20-trino-fest-2023-recap.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+layout: post
+title: "Trino Fest 2023 recap"
+author: "Manfred Moser, Cole Bowden"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2023/trino-fest.png
+---
+
+Last week we held Trino Fest, and we did not find time to chill at the
+lakehouse. Despite the theme of a summercamp for lakehouse analytics, the fest
+kept us all busy. Great demos, amazing announcements, new plugins, and use cases
```
Comment (colebow) [inline _posts/2023-06-20-trino-fest-2023-recap.md:11]: I feel like this is a bit of a negative tone for starting things off. Maybe re-order to, "and it kept us all so busy, we forgot to spend time chilling by the lakehouse!" That feels a little more cheeky and fun IMO.

File: _posts/2023-06-20-trino-fest-2023-recap.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+reached our active audience. Thanks go to our event host and organizer
+[Starburst](https://www.starburst.io/), our sponsors
+[AWS](https://aws.amazon.com/) and [Alluxio](https://www.alluxio.io/), and many
+well-prepared speakers, and our great live audience. Now you get a chance to
```
Comment (colebow) [inline _posts/2023-06-20-trino-fest-2023-recap.md:15]:
```suggestion
[Starburst](https://www.starburst.io/), to our sponsors
[AWS](https://aws.amazon.com/) and [Alluxio](https://www.alluxio.io/), to our many
well-prepared speakers, and to our great live audience. Now you get a chance to
```

File: _posts/2023-06-20-trino-fest-2023-recap.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    </div>
+    <div class="col-sm">
+      <a href="https://aws.amazon.com/">
+        <img src="{{site.url}}/assets/images/logos/aws-small.png" title="AWS, event sponsor">
+      </a>
+    </div>
+  </div>
+</div>
+
+In the weeks running up to the event we had numerous blog posts going out, and
```
Comment (colebow) [inline _posts/2023-06-20-trino-fest-2023-recap.md:40]:
```suggestion
In the weeks leading up to the event, we published numerous blog posts and
```

File: _posts/2023-06-20-trino-fest-2023-recap.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+racked up great interest in the Trino community and beyond. Over 1100
+registrations blew away our numbers from last year. More importantly, during the
+two half days of the event we had up to over 560 attendees watching live and
```
Comment (colebow) [inline _posts/2023-06-20-trino-fest-2023-recap.md:43]:
```suggestion
two half-days of the event, we had over 560 attendees watching live and
```

Comment (colebow) [reply _posts/2023-06-20-trino-fest-2023-recap.md:43]: we can do "up to" or "over" but not both 😛

File: _posts/2023-06-20-trino-fest-2023-recap.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+participating in the busy chat.
+
+## Sessions
+
+And now all those of you, who could not attend every session, or missed out on
```
Comment (colebow) [inline _posts/2023-06-20-trino-fest-2023-recap.md:48]:
```suggestion
If you could not attend every session, or if you missed out on
```

File: _posts/2023-06-20-trino-fest-2023-recap.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+attending completely, have a chance to learn from the presentations and the
```
Comment (colebow) [inline _posts/2023-06-20-trino-fest-2023-recap.md:49]:
```suggestion
attending completely, then we've got great news for you! You still 
have a chance to learn from the presentations and the
```

File: _posts/2023-06-20-trino-fest-2023-recap.md [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+* [<i class="fab fa-youtube" style="color:red;padding-right:0.5em;"/>FugueSQL,
+  Interoperable Python and Trino for interactive
+  workloads](https://youtu.be/aKhI1Phfn-o) presented by [Kevin
+  Kho](https://www.linkedin.com/in/kvnkho/).
+
+## Next up
+
+With this first recap we are sharing all the video recordings with you all. Next
+up are individual blog post for each session that include further useful
+material:
```
Comment (colebow) [inline _posts/2023-06-20-trino-fest-2023-recap.md:109]:
```suggestion
This first recap is sharing all the video recordings with you all if you can't wait.
But stay tuned, because we'll also be publishing individual blog posts and recaps
for each session, and they'll include additional useful info:
```

File: _posts/2023-06-20-trino-fest-2023-recap.md [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+
+* Presentation file to look at your own pace
```
Comment (colebow) [inline _posts/2023-06-20-trino-fest-2023-recap.md:111]:
```suggestion
* Slide decks for you to browse on your own
```

File: _posts/2023-06-20-trino-fest-2023-recap.md [hunk 9]
```diff
@@ -0,0 +1,122 @@
...
+* Video recording
```
Comment (colebow) [inline _posts/2023-06-20-trino-fest-2023-recap.md:112]: Delete this? We're already linking the video recordings here.

File: _posts/2023-06-20-trino-fest-2023-recap.md [hunk 10]
```diff
@@ -0,0 +1,122 @@
...
+  Interoperable Python and Trino for interactive
+  workloads](https://youtu.be/aKhI1Phfn-o) presented by [Kevin
+  Kho](https://www.linkedin.com/in/kvnkho/).
+
+## Next up
+
+With this first recap we are sharing all the video recordings with you all. Next
+up are individual blog post for each session that include further useful
+material:
+
```
Comment (colebow) [inline _posts/2023-06-20-trino-fest-2023-recap.md:110]: Include a line about a summary or recap of the main talking points.

File: _posts/2023-06-20-trino-fest-2023-recap.md [hunk 11]
```diff
@@ -0,0 +1,122 @@
...
+* Presentation file to look at your own pace
+* Video recording
+* Interesting and fun quotes from the speakers and audience
+* Notes and impressions from the audience and event hosts
+* Questions and answer during the event
+* Links to further documentation, tutorials, and other resources
+
+Putting all this together will take us a while, so keep an eye out on our
```
Comment (colebow) [inline _posts/2023-06-20-trino-fest-2023-recap.md:118]:
```suggestion
We'll be rolling out recap posts for a few talks each week, so keep an eye out on our
```
I think "it's gonna take a while" makes it sound like we're incapable of doing work faster. We _are_ capable, we're just choosing to spread the content out to drive more engagement/views on LinkedIn and send a larger volume of traffic to the website over time, rather than in one big burst.

File: _posts/2023-06-20-trino-fest-2023-recap.md [hunk 12]
```diff
@@ -0,0 +1,122 @@
...
+[community chat]({{site.url}}/slack.html) or the website for updates.
+
+At the same time we are already marching ahead with planning towards our next
```
Comment (colebow) [inline _posts/2023-06-20-trino-fest-2023-recap.md:121]:
```suggestion
At the same time, we are already marching ahead and planning our next major
```

File: _posts/2023-06-20-trino-fest-2023-recap.md (hunk 1 above)
Comment (mosabua) [reply _posts/2023-06-20-trino-fest-2023-recap.md:11]: Done

File: _posts/2023-06-20-trino-fest-2023-recap.md (hunk 4 above)
Comment (mosabua) [reply _posts/2023-06-20-trino-fest-2023-recap.md:43]: hahaha

**Review by colebow — APPROVED** [review]
Please make sure you address https://github.com/trinodb/trino.io/pull/459/files#r1235647592, but otherwise LGTM.

**Review by monimiller — APPROVED** [review]
LGTM

**Comment by ryandoowan** [comment]
looks solid!

File: _posts/2023-06-20-trino-fest-2023-recap.md [hunk 13]
```diff
@@ -6,14 +6,13 @@ excerpt_separator: <!--more-->
...
 
-Last week we held Trino Fest, and we did not find time to chill at the
-lakehouse. Despite the theme of a summercamp for lakehouse analytics, the fest
-kept us all busy. Great demos, amazing announcements, new plugins, and use cases
-reached our active audience. Thanks go to our event host and organizer
-[Starburst](https://www.starburst.io/), our sponsors
-[AWS](https://aws.amazon.com/) and [Alluxio](https://www.alluxio.io/), and many
-well-prepared speakers, and our great live audience. Now you get a chance to
-catch up on anything you missed.
+Last week we held Trino Fest, and it kept us all so busy, we forgot to spend
```
Comment (martint) [inline _posts/2023-06-20-trino-fest-2023-recap.md:9]: Unrelated changes for "Remove invalid AWS logos"

File: assets/images/logos/aws-small.png
Comment (martint) [inline assets/images/logos/aws-small.png:1]: What's wrong with the old one?

File: _posts/2023-06-20-trino-fest-2023-recap.md (hunk 13 above)
Comment (mosabua) [reply _posts/2023-06-20-trino-fest-2023-recap.md:9]: Well ... the logo is used in the post and they are separate commits

File: assets/images/logos/aws-small.png
Comment (martint) [reply assets/images/logos/aws-small.png:1]: The new logo looks too big compared to the other ones. The whitespace around it was intentional to keep the size balanced.

File: _posts/2023-06-20-trino-fest-2023-recap.md (hunk 13 above)
Comment (martint) [reply _posts/2023-06-20-trino-fest-2023-recap.md:9]: This commit ("Remove invalid AWS logos") has changes that have nothing to do with removing the logo. Why did this paragraph change? If you look below, there are some other paragraphs with grammar changes that are unrelated to removing the logo.

File: assets/images/logos/aws-small.png
Comment (mosabua) [reply assets/images/logos/aws-small.png:1]: None of the images should have whitespace .. the whitespace is controlled by the css .. we have to fix them all ultimately .. let me take this out for now and send a separate PR later

**Comment by mosabua** [comment]
Somehow this got all messed up, but good news.. all clean now.

**Review by martint — APPROVED** [review]
